### ISSUE TITLE
Smapi will not verify my file location

### ISSUE THREAD

**Issue by redblueatoms** [ZaneYork/SMAPI-Android-Installer#507]
I have followed your written instructions up until it said I pirate a copy of the game through installing a cell phone messaging app to get a file. I can not get smapi to except my file location for mods and I had to go to the trouble of plugging my phone into a laptop to get your file to actually work and be in android/data as I have android 13. 
![Screenshot_20230721-205458](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/140227536/2d7d68e0-1525-4726-84bb-8a004c91e8ad)
![Screenshot_20230721-205444](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/140227536/b863908f-b51b-4c45-bd3f-9ab4afa28abd)
Please give us a full instruction in English that doesn't include weird piracy work arounds that make you download a messaging app to receive a file that for all I know is a virus. I had to unzip the file on my computer and move it into android/data to even get it to be working but smapi doesn't recognize the pathing at all and now it won't open the game it crashes.

**Comment by Zencee** [comment]
hi, your path is incorrect.. just leave it as it is when you install smapi.

![Screenshot_2023-07-22-15-52-43-642_com zane smapiinstaller](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/129819788/66c04b40-2ba1-460a-8c9d-a75bf9f13210)

**Comment by Zencee** [comment]
for now smapi installer cannot access your "data" folder so if you wanna edit your mods you have to use 3rd party apps that can open .json files like quickedit found in playstore

**Comment by redblueatoms** [comment]
Your instructions say this is the path for mods if I can't change the path I can't find the mods so they need to be double installed or something

**Comment by redblueatoms** [comment]
I have followed your suggestion and smapi will not give me a log or install any of my mods it only opens base game

**Comment by Zencee** [comment]
do you have the com.zane.stardewvalley folder in your android/data?

**Comment by redblueatoms** [comment]
Yes, I do.  I went ahead and copied my mods folder from the original place
to this folder as well because it only has saves and screen shots too.

**Comment by Zencee** [comment]
I see.. I think your problem lies on your com.zane folder, it didn't automatically transfer the smapi-internal files when you installed the game. I suggest reinstalling your smapi sdv, while you're at it delete also "Stardew Valley" folder in your base storage as well as "com.zane" folder in the data. Back up your mods/save before reinstalling them. 
![received_1655137651652720](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/129819788/50e9fdab-01b1-4ada-9cf8-dae2647c0245)

**Comment by redblueatoms** [comment]
Currently what my files say should I still delete and start over cause this
will be the eighth time I've had to do that. What all should I delete?

**Comment by redblueatoms** [comment]
Also you have 2 mods folders do they need to be separated for some reason?

**Comment by Zencee** [comment]
just ignore the Mod folder thats for my "to be tested" mods. Once you successfully installed smapi sdv you should have Mods, Saves, Screenshots and smapi-internal folders inside the data/com.zane

**Comment by redblueatoms** [comment]
Okay so I need to save my mods folder and save file which is just a test
save so that doesn't matter and delete everything and start over because I
had to download everything and get my PC to transfer the com.zane file in
the first place because I have android 13

**Comment by Zencee** [comment]
why even use pc though? When you can just do everything in your phone..

**Comment by redblueatoms** [comment]
It's android 13 I tried five different file movers yesturday before
switching to PC and getting it done in three seconds compared to the hours
I spent trying to get the file to move on my phone

**Comment by Zencee** [comment]
damn that phone must be a piece of work then. I've been using zarchiver app to access the data folder. I could give you the exact com.zane folder if you want to, to make your life easier. https://www.mediafire.com/file/9odnwrnrazk8uee/com.zane.stardewvalley.zip/file

**Comment by redblueatoms** [comment]
Thanks it's brand new came out last month the moto g stylus 5g 2023 it's
great except moving files.

**Comment by redblueatoms** [comment]
Okay I did what you said and it's still not showing any of my mods as
installed or even giving the black box pop up of initializes and it's base
game and crashes I don't even see the keyboard button. I'm supposed to have
other mods for changing settings in game too like cbj cheats.

**Comment by redblueatoms** [comment]
Also for some reason it did this the original unpack of the file has to be
done on computer and created a file within a file for some reason and
installing the APK only made a file in the first file with save and
screenshots still no file needed being made like you said. And this is the
third time it's done this.
